core/src/main/scala/geotrellis/server/LayerExtent.scala

```diff
@@ -78,11 +85,11 @@ object LayerExtent extends LazyLogging {
     param: Param
   )(
     implicit reify: ExtentReification[Param],
-             contextShift: ContextShift[IO]
+             contextShift: ContextShift[IO],
+             applicativeError: ApplicativeError[IO, NEL[MamlError]]
   ): (Extent, CellSize) => IO[Interpreted[MultibandTile]] =
     (extent: Extent, cellsize: CellSize) => {
-      val eval = curried(RasterVar("identity"), Interpreter.DEFAULT)
+      val eval = curried(RasterVar("identity"), ConcurrentInterpreter.DEFAULT)
```

Comment:
Worth calling out -- we explicitly shifted to the `ConcurrentInterpreter` everywhere in gt-server, since all of the typeclasses use `IO` as an effect type, and the `NaiveInterpreter` doesn't know anything about `IO`

Reply:
Do we have a sense of what the performance impacts might be? I'm assuming negligible, at worst, but it would be nice to know for sure

Reply:
Should be positive but we don't have load tests here :man_shrugging: